package.json

```diff
@@ -2,15 +2,16 @@
   "name": "@enact/docs-utils",
   "description": "Utilities for parsing and validating JSDoc in Enact projects",
   "version": "0.3.7",
+  "type": "module",
   "engines": {
     "node": ">=10.10.0"
   },
-  "main": "index.js",
+  "main": "index.cjs",
   "bin": {
-    "validate-docs": "index.js"
+    "validate-docs": "index.cjs"
   },
   "scripts": {
-    "lint": "eslint .",
+    "lint": "eslint *js",
```

Comment:
Same here, please share the reason why the extension needs to be changed to `cjs`.

Reply:
There was a warning on my local machine when I run `npm run lint`, but now it's gone (very weird). Anyway, I changed back to `eslint .`

Reply:
OK